### PR TITLE
login page added

### PR DESCRIPTION
**Pull request by Ojusbhutani**

### CODE REVIEW

**Comment by m-e-l-u-h-a-n** [comment]
@Ojusbhutani 
I can't see any changes in the file.

File: pages/login.vue [hunk 1]
```diff
@@ -0,0 +1,194 @@
+<template>
+  <div>
+    <v-container class="fill-height" fluid >
+      <v-row align="center" justify="center">
+        <v-col cols="12" sm="8" md="8">
```
Comment (m-e-l-u-h-a-n) [inline pages/login.vue:5]: only `sm="8"` would have done what you needed

File: pages/login.vue [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+                      </div>
+                      <h4 class="text-center mt-4"></h4>
+                      <v-form>
+                        <v-text-field
+                          label="Email"
+                          name="Email"
+                          prepend-icon="mdi-email"
+                          type="text"
+                          color="white"
+                          v-model="Email"
```
Comment (m-e-l-u-h-a-n) [inline pages/login.vue:35]: #### About all the textfields:
- Add minimum length rules to simple text fields like firstname
- Add a regex validation rule for email
- for password field provide an option to show the password upon user's request.
    - append an icon for toggling display of password.

File: pages/login.vue [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+                          prepend-icon="mdi-lock"
+                          type="password"
+                          color="white"
+                          v-model="password"
+                          required
+                        ></v-text-field>
+                      </v-form>
+                      <div style="text-align:center;">
+                        <v-btn text>
+                          <h4 class="text-center mt-3">Forgot your password</h4>
```
Comment (m-e-l-u-h-a-n) [inline pages/login.vue:51]: No need to add margin to button text, centered text look good.
Add primary color to the button.

File: pages/login.vue [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+                          prepend-icon="mdi-school"
+                          :items="items"
+                          color="white"
+                          v-model="institute"
+                          required
+                        ></v-select>
+                      </v-form>
+                    </v-card-text>
+                    <div class="text-center mt-3">
+                      <v-btn rounded dark>CREATE ACCOUNT</v-btn>
```
Comment (m-e-l-u-h-a-n) [inline pages/login.vue:139]: **Method addition required:**
Upon clicking the sign in and create account button a method should be called which calls a method to check if all the fields are filled correctly and notify users accordingly it is the required even for most basic forms.

**Review by m-e-l-u-h-a-n — CHANGES_REQUESTED** [review]
Great work on the design. :partying_face: :fire: :partying_face: 
Apart from the changes i mentioned in code one more thing need to covered:
#### Current Behaviour
    - In small screen devices the images gets stacked just below the div.
#### Expected Behaviour
    - image should not be visible in small screen devices, instead you could show small text with an alternative option to login or signup with a link.

File: pages/login.vue [hunk 5]
```diff
@@ -31,29 +31,31 @@
...
+                          color="primary"
                         ></v-text-field>
                         <v-text-field
                           id="password"
                           label="Password"
                           name="Password"
                           prepend-icon="mdi-lock"
-                          type="password"
-                          color="white"
+                          :rules="[rules.minlength]"
```
Comment (m-e-l-u-h-a-n) [inline pages/login.vue:44]: I think password's minimum length should be 6 or 8 not zero.

File: pages/login.vue [hunk 6]
```diff
@@ -94,49 +96,64 @@
...
                           type="text"
-                          color="white"
-                          v-model="email"
+                          :rules="[rules.minlength,rules.emailRule]"
+                          v-model="Email"
+                          required
+                          color="primary"
+                        ></v-text-field>
+                        <v-text-field
+                          id="password"
```
Comment (m-e-l-u-h-a-n) [inline pages/login.vue:132]: We need two password fields for signup to confirm the correctness of passwords.

**Review by m-e-l-u-h-a-n — COMMENTED** [review]
@king-11 
For the latest commit following changes are suggested:
- Two password fields in signup page(currently only one).
- password min-length check should be of 6 or 8 and not zero.
So you may consider merging or... implementing these changes before merging.

**Review by m-e-l-u-h-a-n — APPROVED** [review]
This pr can be merged now but has extra previous commits.

**Comment by king-11** [comment]
Will need to rebase it i guess most the commit message aren't pretty clear as to that they do.

**Comment by m-e-l-u-h-a-n** [comment]
@king-11 
Before the latest commit I told him to do interactive rebase but there were options for only those commits which were not pushed to remote. Probably interactive rebase can only be done for commits which have not been pushed. I think we would have to squash and merge from github itself.

**Comment by king-11** [comment]
After pushing it requires force pushing again if we interactive rebase I missed that in the dev talk because, not a good thing to do. But nevermind will do squash and merge

**Pull request merged**